examples/utils/plot_surface2ind_topo.py

```diff
@@ -1,3 +1,10 @@
+"""
+Utils: surface2ind_topo
```

Comment:
If it is easy to change the order (and not have numbers in front of the names - that starts getting cluttered), I would suggest
- Basic Inversion
- Mesh Generation
- Mappings
- Gravity
- Magnetics
- Direct Current Resistivity
- Frequency Domain Electromagnetics
- Natural Source Electromagnetics (instead of Magnetotellurics)
- Time Domain Electromagnetics
- Fluid Flow
- Utilities

(so basics, applications in ~ order of complexity / grouped with like things, utilities)